src/__experimental__/components/textarea/textarea.stories.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  boolean, number, text, select
+} from '@storybook/addon-knobs';
+import { State, Store } from '@sambego/storybook-state';
+import OptionsHelper from '../../../utils/helpers/options-helper';
+import Textarea from '.';
+import { notes, info } from './documentation';
+
+const store = new Store({
+  value: ''
+});
+
+const handleChange = ({ target: { value } }) => {
+  store.set({ value });
+};
+
+const rangeOptions = {
+  range: true,
+  min: 0,
+  max: 300,
+  step: 1
+};
+
+storiesOf('Experimental/Textarea', module)
+  .addParameters({
+    info: {
+      propTablesExclude: [State]
+    }
+  }).add(
+    'default',
+    () => (
+      <State store={ store }>
+        <Textarea
+          onChange={ handleChange }
+          { ...getTextareaStoryProps() }
+        />
+      </State>
+    ),
+    {
+      info: { text: info },
+      notes: { markdown: notes }
+    },
+  );
+
+function getTextareaStoryProps() {
```

Comment:
that method is redundant as it will not be reused

Reply:
Removed, thanks